server.js

```diff
@@ -2,65 +2,40 @@
 const express = require('express');
 const app = express();
 
-// run `npm install` to install dependencies in package.json
+const MarsMissions = require('./models/missions');
 
 // * Your mission is to complete the app
 // * The app will need routes for index and show
+
+//Index route - missions
+app.get('/missions', (req, res) => {
+  res.send(MarsMissions);
+});
+
+//show route - missions
+app.get('/missions/:id', (req, res) => {
+  res.render('show.ejs', {
```

Comment:
to get this to work, your views need to be in a directory callled `views` instead of the `missions` directory they are currently in.